# HG changeset patch
# Parent 7528376bbc7bdeae642e31cba2bdb3e52930c64b
Fixup pf start/stop messages on the console, Enabling pfpf enabled

diff -r 7528376bbc7b etc/rc.d/pf
--- a/etc/rc.d/pf
+++ b/etc/rc.d/pf
@@ -25,7 +25,7 @@
 
 pf_start()
 {
-	check_startmsgs && echo -n 'Enabling pf'
+	check_startmsgs && echo -n 'Enabling pf: '
 	$pf_program -F all > /dev/null 2>&1
 	$pf_program -f "$pf_rules" $pf_flags
 	if ! $pf_program -s info | grep -q "Enabled" ; then
@@ -37,7 +37,7 @@
 pf_stop()
 {
 	if $pf_program -s info | grep -q "Enabled" ; then
-		echo -n 'Disabling pf'
+		echo -n 'Disabling pf: '
 		$pf_program -d
 		echo '.'
 	fi
